base-isa.md

```diff
@@ -108,26 +106,24 @@ Undefined control registers are reserved and reading from or writing to them is
 
 ## Memory Semantics
 
-Unaligned memory accesses are undefined behavior. A memory access is unaligned if the address modulus the read/write width is not equal to zero. The read/write width is defined by the SS bits, which for the base ISA corresponds to a read/write width of 2 bytes.
+Unaligned memory accesses are _unspecified_ behavior. A memory access is unaligned if the address modulus the read/write width is not equal to zero. The read/write width is defined by the SS bits, which for the base ISA corresponds to a read/write width of 2 bytes.
```

Comment:
This is another sentence that I think benefits from restricting context to base-isa. "A memory access is unaligned if the address is not divisible by 2." The `SS` extensions could be modified to mention how they affect alignment requirements if they don't already.

Reply:
Done, please take a look

Reply:
LGTM